site/layouts/partials/features.html

```diff
@@ -2,15 +2,24 @@ <h2>Features</h2>
 
 <dl>
   <dt>MongoDB Driver</dt>
-  <dd>A brand new MongoDB driver for Node.js that keeps compatibility
-	  with the 1.4.x driver series, only breaking behavior where the
-	  1.4.x branch had significant problems. The driver also includes
-	  support for the shared CRUD API specification and the Server
-	  Discovery and Monitoring Specification (SDAM).</dd>
-  <dt>Core driver</dt>
-  <dd>The MongoDB Node.js core driver is the new underpinning for the
-	  driver and is meant for framework developers that do not need the
-	  helpers and abstractions available in the full driver. This driver
-	  is not meant for end users, as it offers none of the conveniences
-	  of the higher level APIs</dd>.
+  <dd>
+	Whether you are new to MongoDB or you just want to brush up on the core concepts, you can check out the Fundamentals section which features material on using the Node.js driver for the following:
+	<ul>
+		<li><a href="/drivers/node/fundamentals/connection">Connection Guide</a>: connect to a MongoDB
```

Comment:
Don't these need to point to off-domain to `https://docs.mongodb.com/drivers/node/`?

Reply:
Ah good catch, did some reformatting and lost the full links, fixed!